### PR TITLE
Remove a trailing comma which can cause issues

### PR DESCRIPTION
**Pull request by mircobabini**
### All Submissions:

* [x] Have you followed the [Contributing guideline](https://github.com/strangerstudios/paid-memberships-pro/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/strangerstudios/paid-memberships-pro/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Trailing commas in functions call are valid since PHP 7.3. We are 5.6+ compatible, then we should avoid it.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: adminpages/orders.php [hunk 1]
```diff
@@ -1711,13 +1711,14 @@ function pmpro_ShowMonthOrYear() {
 							<a href="<?php echo esc_url( add_query_arg( array( 'page' => 'pmpro-orders', 's' => $order->subscription_transaction_id ), admin_url( 'admin.php' ) ) ); ?>" title="<?php esc_attr_e( 'View all orders for this subscription', 'paid-memberships-pro' ); ?>" class="pmpro_order-renewal"><?php esc_html_e( 'Renewal', 'paid-memberships-pro' ); ?></a>
 						<?php } ?>
 					</td>
-					<td class="column-date">
+					<td class="
```
Comment (andrewlimaza) [inline adminpages/orders.php:1714]: This HTML class was accidentally removed.

Comment (andrewlimaza) [reply adminpages/orders.php:1714]: Other than that the PR looks good.

Comment (mircobabini) [reply adminpages/orders.php]: Fixed by cfc7b14322a038fb16139fea189205e281001fb5. Sorry.

Comment (andrewlimaza) [reply adminpages/orders.php]: No problem! These things happens :) moved this to ready for merge.

**Comment by ideadude** [comment]
Nice catch. Merging. Will go out in 2.8.2

**Pull request merged**